apps/src/dance/ai/DanceAiModal.tsx

```diff
@@ -461,8 +444,13 @@ const DanceAiModal: React.FunctionComponent<DanceAiProps> = ({onClose}) => {
                       !processingDone &&
                       moduleStyles.beamImageVisible
                   )}
+                  alt="A.I. Bot Beam"
```

Comment:
We officially refer to the bot as just "A.I.", rather than "A.I. Bot".  That said, we might want to be more descriptive for screenreaders, in which case either "bot" or "robot" could be reasonable.

Reply:
That said, I don't think the user would ever focus on the bot itself, so do we need alt text for it?

Reply:
Oh yep sure, I can change to "A.I.". I added this because occasionally when the screen reader was not focused on a button in the modal (ex. during mode transitions) it would start to read out the content of all elements, and without alt text it was reading back a really long asset file name.

Reply:
Renamed to "A.I." for now - figured we can update these down the road if needed

Reply:
Thanks.  

Incidentally, should we prep these strings for localization?  And are there potentially other alt text strings?

Reply:
Oh yep good call - didn't want to clash with https://github.com/code-dot-org/code-dot-org/pull/54619, but I can prep these for localization if that would be easier to merge for you?

Reply:
Oh, I can just add them to https://github.com/code-dot-org/code-dot-org/pull/54619 for now.  We just need to merge that by itself by the end of the week, and we can prep their usage in code separately next week.

Reply:
If you think of any others I should add, let me know and I can add them too.

Reply:
Sounds good, I think these should be it for now!